src/java.base/share/classes/java/net/Inet4Address.java

```diff
@@ -134,6 +137,64 @@ class Inet4Address extends InetAddress {
         holder().originalHostName = hostName;
     }
 
+    /**
+     * Creates an {@code Inet4Address} based on the provided textual representation of
+     * an IPv4 address.
+     * <p>The following IPv4 address {@linkplain Inet4Address##format
+     * textual representations} are supported by this method:
+     * {@snippet :
+     *  // Dotted-decimal 'd.d.d.d' form with four part address literal
+     *  Inet4Address.ofLiteral("7.08.9.010") ==> /7.8.9.10
+     *
+     *  // Dotted-decimal 'd.d.d' form with three part address literal,
+     *  // the last part is placed in the right most two bytes
+     *  // of the constructed address
+     *  Inet4Address.ofLiteral("127.0.257") ==> /127.0.1.1
+     *
+     *  // Dotted-decimal 'd.d' form with two part address literal,
+     *  // the last part is placed in the right most three bytes
+     *  // of the constructed address
+     *  Inet4Address.ofLiteral("127.257") ==> /127.0.1.1
+     *
+     *  // 'd' form with one decimal value that is stored directly in
+     *  // the constructed address bytes without any rearrangement
+     *  Inet4Address.ofLiteral("02130706689") ==> /127.0.1.1
+     * }
+     * <p>If the provided address literal cannot represent a valid IPv4 address an
```

Comment:
I think this behavior is the same as existing methods, except that only the decimal format is supported? I wonder is it really necessary to describe it here, as it it would be quite an arcane usage, and the section in the class doc "Textual representation if IP addresses" already partially covers it. The snippet example with the leading zero seems to be implying that octal format is not supported, but without really saying it out loud. 

Maybe the apidoc for the method could just say that the decimal formats are the only ones supported, but otherwise refer to the class docs for how the different variants work ? And maybe move the snippet there as well?